force-app/main/default/classes/Data Recipes/SOQLRecipes.cls

```diff
@@ -21,13 +20,11 @@ public inherited sharing class SOQLRecipes {
     @SuppressWarnings('PMD.ApexCRUDViolation')
```

Comment:
I thought the latest version of the Salesforce CLI Scanner does not flag these queries anymore. See if it does not, maybe we remove this.

Reply:
it is here...